Makefile

```diff
@@ -0,0 +1,98 @@
+# Copyright (c) 2020 CUJO LLC. All rights reserved.
```

Comment:
License would be good yes, and then we'll need to change these notices as well. I think MIT is what we should be putting it under, @james-mathews-cujo can you confirm?